src/raster/r.in.usgs/r.in.usgs.py

```diff
@@ -144,12 +146,47 @@
 from multiprocessing import Process, Manager
 import json
 import atexit
+from pathlib import Path
 
+import grass.script as gs
 from grass.exceptions import CalledModuleError
 
 cleanup_list = []
 
 
+def get_cache_dir(name):
+    """Get the default user cache directory
+
+    The name parameter is used to distinguish cache data from different
+    componenets, e.g., from different modules.
+
+    The function creates the directory (including all its parent directories)
+    if it does not exist.
+    """
+    app_version = gs.version()["version"]
+    if sys.platform.startswith("win"):
+        # App name, directory, and the assumption that the directory exists
+        # (and thus that it should be used) are derived from the startup script.
+        app_name = "GRASS8"
+        path = Path(os.getenv("APPDATA")) / app_name / "Cache" / name
```

Comment:
Copy-paste from grass.py, but now done in a better way since I have the version string already and there is no link with explicit link with grass.py anyway. (This whole function is a library function candidate.)